application/classes/Controller/Api/Posts/Bulk.php

```diff
@@ -0,0 +1,79 @@
+<?php defined('SYSPATH') OR die('No direct access allowed.');
+
+/**
+ * Ushahidi API Posts Bulk Controller
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application\Controllers
+ * @copyright  2013 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Core\Usecase;
+
+class Controller_Api_Posts_Bulk extends Ushahidi_Rest {
+
+	/**
+	 * @var int Post Parent ID
+	 */
+	protected $_parent_id = NULL;
+
+	/**
+	 * @var string Post Type
+	 */
+	protected $_type = 'report';
+
+	protected function _resource()
+	{
+		return 'posts_bulk';
+	}
+
+	// Ushahidi_Rest
+	protected function _scope()
+	{
+		return 'posts_bulk';
```

Comment:
I think scope should still just be posts .. we don't really need another oauth scope just for this.